lib/src/story.dart

```diff
@@ -19,14 +28,16 @@ class Property<T> {
 class ListProperty<T> extends Property<T> {
   final List<T> list;
 
-  ListProperty(String name, T defaultValue, this.list)
-      : super(name, defaultValue);
+  ListProperty(String name, T defaultValue, this.list,
+      {ControlProperty? controlProperty})
+      : super(name, defaultValue, controlProperty: controlProperty);
 }
 
 class DashbookContext {
   Map<String, Property> properties = {};
 
-  String textProperty(String name, String defaultValue) {
+  String textProperty(String name, String defaultValue,
+      {ControlProperty? controlProperty}) {
```

Comment:
**chore (blocking):** we need to utilize the `controlProperty` here and pass it to the `Property` like we added for the other property types.
https://github.com/erickzanardo/dashbook/blob/0de95e01ae6898d27deb4e5f0b7fedcae763fad1/lib/src/story.dart#L33-L34

```dart
      final property = Property<String>(name, defaultValue,
          controlProperty: controlProperty);
      properties[name] = property;
```